card/mod/05_standard/set/type/search_type.rb

```diff
@@ -231,13 +250,6 @@ def default_search_params
     set_default_search_params :default_limit=>20
   end
   
-  
 end
 
 
```

Comment:
Right.  In many cases, all we were doing to support bootstrap was adding CSS classes.  Sometimes it was hard to do that without overriding an entire view, so we temporarily did that, but it was pretty ugly to have both, with the overridden one never used.  I do hope we improve the api to make it so that adding in CSS classes with super is a very easy thing to do; it would be a very common use case for many integrations.

There were, however, some cases where we changed the HTML, and our strategy there has been for that HTML to become the new default.  Examples of that are the new card-header, the new page header, and the new handling of form fields.  We kept the changes as minimal as possible, but I do suspect there will be some old skins that will need to be updated if folks choose to update to the new bootstrap layout.